src/WebJobs.Script/Workers/Rpc/Configuration/RpcWorkerConfigFactory.cs

```diff
@@ -32,22 +32,19 @@ internal class RpcWorkerConfigFactory
                                         ILogger logger,
                                         ISystemRuntimeInformation systemRuntimeInfo,
                                         IEnvironment environment,
-                                        IMetricsLogger metricsLogger)
+                                        IMetricsLogger metricsLogger,
+                                        IWorkerProfileManager workerProfileManager)
         {
             _config = config ?? throw new ArgumentNullException(nameof(config));
             _logger = logger ?? throw new ArgumentNullException(nameof(logger));
             _systemRuntimeInformation = systemRuntimeInfo ?? throw new ArgumentNullException(nameof(systemRuntimeInfo));
             _environment = environment ?? throw new ArgumentNullException(nameof(environment));
-            _metricsLogger = metricsLogger;
-            _workerRuntime = _environment.GetEnvironmentVariable(RpcWorkerConstants.FunctionWorkerRuntimeSettingName);
-
-            var conditionProviders = new List<IWorkerProfileConditionProvider>
-            {
-                new WorkerProfileConditionProvider(_logger, _environment)
-            };
+            _metricsLogger = metricsLogger ?? throw new ArgumentNullException(nameof(metricsLogger));
+            _profileManager = workerProfileManager ?? throw new ArgumentNullException(nameof(workerProfileManager));
 
-            _profileManager = new WorkerProfileManager(_logger, conditionProviders);
+            _workerRuntime = _environment.GetEnvironmentVariable(RpcWorkerConstants.FunctionWorkerRuntimeSettingName);
 
+            string assemblyLocalPath = Path.GetDirectoryName(new Uri(typeof(RpcWorkerConfigFactory).Assembly.CodeBase).LocalPath);
```

Comment:
Is this used for anything?

Reply:
No, I actually don't even remember adding this line at all ☠️ thanks for catching